### ISSUE TITLE
Add proper cleanup

### ISSUE THREAD

**Issue by PVince81** [owncloud-archive/acceptance-testing#38]
Currently after each scenario the next scenario runs with a "dirty" state from the previous one.

I search for documentation online but it seems the most common pattern is to connect directly to the DB and reset the data there. That would be one possible approach.

Another approach, which is probably more complex, is to create steps that will cleanup over the UI (for example delete a user after the creation test, etc).

We should probably look into direct DB access first as it might help accelerate some tests. For example "Given that user user1 exists" could directly add an entry for that (which seems to be the way how others do it)

**Comment by Gomez** [comment]
Yes, we should reset the DB after every Scenario. This could be done with a chef command and executed after every Scenario.
